hyperliquid/info.py

```diff
@@ -526,3 +526,8 @@ def unsubscribe(self, subscription: Subscription, subscription_id: int) -> bool:
 
     def name_to_asset(self, name: str) -> int:
         return self.coin_to_asset[self.name_to_coin[name]]
+
+    def close(self):
+        if not hasattr(self, "ws_manager") or self.ws_manager is None:
```

Comment:
Please use the same if/else logic as above for handling existence of self.ws_manager. In this case, we would want to throw an exception so that we don't silently proceed when unintentionally calling this function.